app/extensions/brave/content/scripts/flashListener.js

```diff
@@ -20,18 +22,18 @@ function isAdobeLink (href) {
       }
     })
   }
-  var observer = new window.MutationObserver(function (mutations) {
+  setTimeout(() => {
+    var observer = new window.MutationObserver(function (mutations) {
+      replaceAdobeLinks()
+    })
     replaceAdobeLinks()
```

Comment:
it definitely helps to defer it and I don't think anyone will really notice the 1s delay